doc/README_Source.md

```diff
@@ -2,144 +2,168 @@
 
-Data will always be read from Cosmos DB as JSON. 
+```bash
 
-The *key.converter* and *value.converter* configuration should be set according to how you want the data serialized when written to the Kafka topic. 
+# clone the kafka-connect-cosmosdb repo if you haven't done so already
+git clone https://github.com/microsoft/kafka-connect-cosmosdb.git
+cd kafka-connect-cosmosdb
 
-If the data in Cosmos DB contains the schema embedded in the document and it is in the following format - 
+# package the source code into a JAR file
+mvn clean package
 
-```javascript
+# include the following JAR file in Confluent Platform installation
+ls target/*dependencies.jar
 
 ```
 
-then you can configure the value.converter to use JSON with Schema by setting the following configuration: 
+For more information on installing the connector manually, refer to these [instructions](https://docs.confluent.io/current/connect/managing/install.html#install-connector-manually).
 
-```properties
+### Create Kafka topic
+Create a Kafka topic using Confluent Control Center. For this quickstart, we will create a Kafka topic named `apparels` and will write JSON data (non-schema embedded) to the topic.
 
-```
+To create a topic inside Control Center, see [here](https://docs.confluent.io/platform/current/quickstart/ce-docker-quickstart.html#step-2-create-ak-topics).
 
-It is possible to have the Source connector output CSV string by using StringConverter as follows: 
+### Create the source connector
 
-```properties
-```
+#### Create the Cosmos DB Source Connector in Kafka Connect
 
-## Quick Start
+The following JSON body defines the config for the Cosmos DB Source Connector. You will need to fill out the values for `connect.cosmosdb.connection.endpoint` and `connect.cosmosdb.master.key`, which you should have saved from the [Cosmos DB setup guide](./CosmosDB_Setup.md).
 
-### Prerequisites
-* [Event Hub](https://docs.microsoft.com/pt-br/azure/event-hubs/event-hubs-for-kafka-ecosystem-overview)
-* [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli) (requires separate installation)
+Refer to the [source properties](#source-configuration-properties) section for more information on each of these configuration properties.
 
-### Create Azure Cosmos DB Instance, Database and Collection
+```json
+{
+  "name": "cosmosdb-source-connector",
+  "config": {
+    "connector.class": "com.microsoft.azure.cosmosdb.kafka.connect.source.CosmosDBSourceConnector",
+    "tasks.max": "1",
+    "key.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "connect.cosmosdb.task.poll.interval": "100",
+    "connect.cosmosdb.connection.endpoint": "https://<cosmosinstance-name>.documents.azure.com:443/",
+    "connect.cosmosdb.master.key": "<cosmosdbprimarykey>",
+    "connect.cosmosdb.databasename": "kafkaconnect",
```

Comment:
change this to something like _<replace this with your cosmos db master key>_
or, add a note under this section to tell people to replace the placeholders. 
otherwise, we end up with a bunch of people copying & pasting the config as is and then wondering why the connector fails.

Reply:
I had a note up there, but explicitly mentioned which properties to updated before configuring this connector